p2p/protocols/protocol.go

```diff
@@ -359,7 +303,7 @@ func (p *Peer) Send(ctx context.Context, msg interface{}) error {
 
 	code, found := p.spec.GetCode(msg)
 	if !found {
-		return errorf(ErrInvalidMsgType, "%v", code)
+		return fmt.Errorf("Invalid message type %v ", code)
```

Comment:
Errors should not start with a capital letter. This goes for all error messages below, also.

Reply:
I agree, I did not change this yet, as most of the messages was like this, wanted to brainstorm about the errors first and then change, but I think I have enough info now in the comments about the different errors :)